02-getting_started.Rmd

```diff
@@ -1,67 +1,249 @@
+
+3.  Create the build directory in the `BUILDDIR`:
+
+        export BUILDDIR="$HOME/bin/R"
+        mkdir -p $BUILDDIR
+        cd $BUILDDIR
+        
+4.  Configure the R installation (with `--enable-R-shlib` so that RStudio IDE can use it):
+        
+        $TOP_SRCDIR/configure --enable-R-shlib
+        
+4.  Build R :
+
+        make 
+
+
+5.  Check that R works as expected:
+
+        make check
+
+    There are other checks you can run:
+
+        make check-devel
+        make check-recommended
+
+
+[^02-getting_started-2]: Recommended packages are not in the subversion repository.
+
+
+If we don't want to build R in a different directory than the source code we can simply use:
+
+    cd $TOP_SRCDIR 
+    svn update
+    tools/rsync-recommended
+    $TOP_SRCDIR/configure  --enable-R-shlib
+    make 
+    make check
+
+Once you successfully build R from source you can modify R source code to fix an issue: Prepare a patch (See [this guide](https://www.r-project.org/bugs.html#how-to-submit-patches)) and after checking that R works as intended (`make check-devel`) submit the patch for the R core consideration.
+(See the [lifecycle of a patch](#FixBug) chapter).
+
+
+To use the r-devel version in RStudio to you can do the following:
+
+    export RSTUIDO_WHICH_R="$BUILDDIR/bin/R"
+    cd $TOP_SRCDIR
+    rstudio
+
+
+### Windows
+
+#### Binaries
+
+The latest documentation about installing R and packages from source at the current time can be find [here](https://cran.r-project.org/bin/windows/base/howto-R-4.2.html).
+
+1.  The binary builds of R for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/).
+    Along with the link to the latest stable release, this page also contains links to the binary builds of `r-patched` and `r-devel`.
+
+2.  Click on the download links to download an executable installer.
+
+3.  Select the language while installing, read the GNU general public license information, and select the destination location to start the installation.
+    You will be prompted to select components at this stage: `User installation`, `64-bit User installation`, or `Custom installation`.
+    The default option may be chosen for the questions from this step onwards to complete the installation.
+    
+You can also use the [binaries for `r-devel` made available](https://cran.r-project.org/bin/windows/base/rdevel.html) which are not so up to date. 
+
+#### From source
+
+You can download the R sources on Windows <https://svn.r-project.org/R/>. 
+But if you want to keep up with changes it is recommended to checkout the sources from svn, it is probably best to install an SVN client.
+Either TortoiseSVN (<https://tortoisesvn.net/>, command line tool, and Windows Explorer integration) or SlikSVN (<https://sliksvn.com/download/>, just the command line tool) is recommended.
+
+1.  Open the Rtools42 terminal, and update and get the tools:
+
+      pacman -Syuu
+      pacman -Sy wget subversion
```

Comment:
We probably don't need subversion here if we are using TortoiseSVN or SlikSVN. I think it is helpful to recommend people install Subversion outside of the MSYS shell, so that it can be found more easily by other tools, e.g. RStudio.